concent_api/requirements.txt

```diff
@@ -2,7 +2,7 @@
 assertpy
 Django >= 1.11, < 1.12
 psycopg2
--e git://github.com/golemfactory/golem-messages.git@v2.11.0#egg=golem_messages
+-e git://github.com/golemfactory/golem-messages.git@f120ac4823cf1b0037ea4077e14e191102594a79#egg=golem_messages
```

Comment:
Why is a commit hard-coded here?

Reply:
As I wrote in PR comment in the top 
>About requirements:
Temporary solution before new version of Golem Messages will tagged.

I need to wait with merging it till golem_messages won't have new tag.